code/controllers/subsystem/monitor.dm

```diff
@@ -139,3 +146,29 @@ SUBSYSTEM_DEF(monitor)
 		stalemate = TRUE
 	else
 		stalemate = FALSE
+
+/// If xeno are losing AND they have a lot of burrowed, give xeno bonuses
+/datum/controller/subsystem/monitor/proc/balance_xeno_team()
+	var/datum/job/xeno_job = SSjob.GetJobType(/datum/job/xenomorph)
+	if(!balance_system_authorised || current_state >= STATE_BALANCED || (xeno_job.total_positions-xeno_job.current_positions) <=  length(GLOB.alive_xeno_list) * TOO_MUCH_BURROWED_PROPORTION)
```

Comment:
consider moving this balance_system_authorised check to outside this proc to the automated process so admins can proccall this if they want to do it for a oneoff if its off but should be balanced

If you do it rename balance_system_authorised to something like is_enabled_autobalance